src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+
+    RED,
+    GREEN,
+    BLUE,
+    YELLOW,
+    value;
```

Comment:
The enum value 'value' is not named according to the convention. Enum values should be in uppercase letters. If 'value' is not an intended enum constant, please remove or rename it to fit the context of 'Colors' enum.